packages/core-plugin-api/src/extensions/extensions.tsx

```diff
@@ -262,3 +294,13 @@ export function createReactExtension<
     },
   };
 }
+
+function createPermission(
+  plugin: BackstagePlugin<any, any>,
+  name?: string,
+): Permission {
+  return {
+    name: [plugin.getId(), name || 'default'].join('.'),
```

Comment:
Can I propose we make the name fixed, and use attributes to reflect the plugin ID and route? I think that will make it easier for policy authors to match these permissions.